pkg/googleworkspace/service.go

```diff
@@ -176,8 +179,10 @@ func getTokenFromWeb(config *oauth2.Config) *oauth2.Token {
 
 	m := http.NewServeMux()
 	// TODO: remove hardcoded port.
-	s := http.Server{Addr: ":9999", Handler: m}
-	config.RedirectURL = "http://localhost:9999/callback"
+	// s := http.Server{Addr: ":9999", Handler: m}
+	// config.RedirectURL = "http://localhost:9999/callback"
+	s := http.Server{Addr: ":9000", Handler: m}
+	config.RedirectURL = "https://doc-vault.dev.razorpay.in/callback"
```

Comment:
should be passed as environment variable ?

Reply:
they have not provided this facility, we might add it, but actually, it is not required as we have now shifted to **service account**, this part of the `code `was its alternative.